lib/ace/mode/latex_worker.js

```diff
@@ -737,10 +755,15 @@ var InterpretTokens = function (TokeniseResult, ErrorReporter) {
                 // skip over filenames, may contain _
                 newPos = read1filename(TokeniseResult, i);
                 if (newPos === null) { continue; } else {i = newPos;};
-            } else if (seq === "hbox" || seq === "text" || seq === "mbox" || seq === "footnote" || seq === "intertext" || seq === "shortintertext" || seq === "textnormal" || seq === "tag" || seq === "reflectbox" || seq === "textrm") {
+            } else if (seq === "hbox" || seq === "text" || seq === "mbox" || seq === "footnote" || seq === "intertext" || seq === "shortintertext" || seq === "textnormal" || seq === "reflectbox" || seq === "textrm") {
```

Comment:
It seems like this test for `seq === 'tag'` will override the later one below with the `readOptionalStar` - how does it work with the `\tag*` in that case - maybe only the case below is needed?

Reply:
Ah yes, I see the `tag` was already there and you removed it.